src/game.cpp

```diff
@@ -2983,15 +3023,12 @@ void Game::decreaseViewRange()
 
 void Game::toggleFullViewRange()
 {
-	static const wchar_t *msg[] = {
-		L"Disabled full viewing range",
-		L"Enabled full viewing range"
-	};
-
 	draw_control->range_all = !draw_control->range_all;
-	infostream << msg[draw_control->range_all] << std::endl;
-	m_statustext = msg[draw_control->range_all];
 	runData.statustext_time = 0;
+	if (draw_control->range_all)
+		showStatusTextSimple("Enabled full viewing range");
```

Comment:
As discussed elsewhere this does need changing, as 'full view range' implies a normal in-game view range, which encourages players to use this mode when they should not.
This needs to describe 'view entire world' or 'unlimited view range' or something that discourages this mode being used in gameplay.

Reply:
"Enabled view as far as you can" (?)